app.js

```diff
@@ -5,7 +5,7 @@ const logger = require('morgan');
 const { databaseInit } = require('./src/mongo/initDb');
 const errorHandler = require('./src/middleware/errorHandler');
 
-const usersRouter = require('./routes/users');
```

Comment:
I think this folder and file are still in the project, if not using them, we can delete them.

Reply:
Also I Think having routes and Controllers separate could add some structure and better organization. I know the app is small now, but when it gets bigger, having this separation might improve readability.

So I think Route/users could have 
`router.get('/', getUser)` and `router.get('/:id', getUserById)` and the controller/users will contain getUser and getUserById

Reply:
Good call.  Deleted the old route file.  Will refactor it later when I get a better idea of the overall footprint of the APIs